.eslintrc

```diff
@@ -22,6 +22,6 @@
   },
 
   "rules": {
-    "comma-dangle": ["always-multiline"]
+    "comma-dangle": [2, "always-multiline"]
```

Comment:
Happy for you to fix them :)

On Fri, 27 May 2016 16:25 isabelrios, <notifications@github.com> wrote:

> In .eslintrc
> <https://github.com/mozilla-magnet/magnet-client/pull/116#discussion_r64922156>
> :
>
> > @@ -22,6 +22,6 @@
> >    },
> >
> >    "rules": {
> > -    "comma-dangle": ["always-multiline"]
> > +    "comma-dangle": [2, "always-multiline"]
>
> Great, that worked! There are still a few lint errors due to unused or non
> define vars in different files previous to this PR
> @wilsonpage <https://github.com/wilsonpage> what do you prefer me to do,
> fixing them in this PR or in another (to not create a big mixed PR)
> Thanks!
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla-magnet/magnet-client/pull/116/files/9ee0a03b8ea017f581e6db56159a378352aefb5a..ee3c3b064c5607f78762c196fb9cfb6872fcc69e#r64922156>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA-Shwi1_ppHARpPq4J7ao64cY88RT5Mks5qFwzwgaJpZM4IoUqO>
> .
>